features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -339,6 +342,16 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
     const SUPPORT_PKCE_PLAIN_ALGORITHM_VALUE: string = "supportPlainTransformAlgorithm";
     const JWT: string = "JWT";
 
+    const featureConfig: FeatureAccessConfigInterface = useSelector((state: AppState) => {
+        return state.config?.ui?.features?.applications;
+    });
+
+    const IsHybridFlowAppLevelConfigEnabled: boolean = useMemo(() => {
+        const disabledFeatures: string[] = featureConfig?.disabledFeatures;
+
+        return !disabledFeatures.includes("applications.hybridFlow");
+    }, [ featureConfig ]);
```

Comment:
Use the readymade functions in core modules instead.

Ref: https://github.com/wso2/identity-apps/blob/master/docs/FEATURE_FLAGS.md#:~:text=%5D%0A%20%20%7D%0A%7D-,Use%20Feature%20Flags%20in%20Your%20Code,-Finally%2C%20use%20the

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6709/commits/05f71e055f845e30333310622a50976081a25647